### PR TITLE
feat(portal): Portal customization

### PR DESCRIPTION
**Pull request by lauthieb**
Closes gravitee-io/issues#596

### CODE REVIEW

File: constants.json [hunk 1]
```diff
@@ -1,7 +1,8 @@
 {
-    "baseURL": "/management/",
+    "baseURL": "https://nightly.gravitee.io/management/",
```
Comment (aelamrani) [inline constants.json]: Be careful, this must not be part of the bundle.

FYI, you can now execute the UI pointing on nightly rest-api by executing : `npm run serve:nightly`

**Comment by lauthieb** [comment]
Ready to merge :)

**Comment by brasseld** [comment]
There is an issue with this PR.
Initially, background-color of section where you can read "Unleash the power of..." must be white.

capture d ecran 2017-05-31 a 22 53 46

**Comment by lauthieb** [comment]
Ok, I have fixed it

**Pull request merged**